log/development.log

```diff
@@ -5,985 +5,730 @@ Processing by Rails::WelcomeController#index as HTML
 Completed 200 OK in 40ms (Views: 23.5ms | ActiveRecord: 0.0ms)
 
 
-  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
```

Comment:
Can you untrack this file please. It is not necessary as everyone will have their own and it will cause conflicts everytime.

Reply:
Yes, I tried to adding it to the `.gitignore` file but it doesn't work. I'll keep trying more alternatives.